scripts/runtests.sh

```diff
@@ -89,8 +89,8 @@ validate () {
                 echo "user not assigned with the expected group sudo"
                 exit 1
             fi
-            lang="$(grep -Eo 'LANG="([^.@ _]+)' .subiquity/etc/default/locale | cut -d \" -f 2)"
-            if [ -z "$( ls .subiquity/var/cache/apt/archives/) | grep $lang" ] ; then
+            lang="$(grep -Eo 'LANG=([^.@ _]+)' .subiquity/etc/default/locale | cut -d= -f 2-)"
```

Comment:
I'd keep the quote removal in the end. I'm not quite sure under what circumstances `update-locale` quotes the values it writes to the `/etc/default/locale` file. In my system file it has quotes, under the integration tests it seems not to add them. I'd rather follow your approach with the following addition:
```suggestion
            lang="$(grep -Eo 'LANG=([^.@ _]+)' .subiquity/etc/default/locale | cut -d= -f 2- | cut -d\" -f 2-)"
```

In case of absent quotes that wouldn't fail anyway, so we should be good to go. Other than that, LGTM.

Reply:
Good point, I didn't realize `cut` would print the string if no delimiter is found.
Thanks!